### PR TITLE
Adding Windows Code Signing to GitHub Actions 

### PR DESCRIPTION
**Pull request by joseph-flinn**
## Summary
Moving code signing into the Github Actions by hosting our signing cert in Azure. Moving all release specific work into a Release pipeline that will be manually triggered (as a side benefit, this sets us up for easy semi-automatic version bumps in the future)

## Files Changed
- .github/workflows/release.yml
- .github/workflow/build.yml
- package.json
- sign.js
- appveyor.yaml

## ⚠️ Notes
The signing secrets below can be found in the DevOps Collection

Repo Secrets Needed:
- SIGNING_VAULT_URL
- SIGNING_CLIENT_ID
- SIGNING_CLIENT_SECRET
- SIGNING_TENANT_ID
- SIGNING_CERT_NAME
- GITHUB_TOKEN

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -196,7 +197,7 @@
       "applicationId": "bitwardendesktop",
       "identityName": "8bitSolutionsLLC.bitwardendesktop",
       "publisher": "CN=14D52771-DE3C-4886-B8BF-825BA7690418",
-      "publisherDisplayName": "8bit Solutions LLC",
+      "publisherDisplayName": "Bitwarden Inc",
```
Comment (kspearrin) [inline package.json:200]: our publisher display name with windows store is still 8bit solutions llc

**Review by kspearrin — COMMENTED** [review]
Why are there several things commented out in the build.yml?

File: package.json (hunk 1 above)
Comment (joseph-flinn) [reply package.json]: Ah, I hadn't removed any of my testing code yet. Thanks for that catch!

**Review by kspearrin — APPROVED** [review]

**Comment by joseph-flinn** [comment]
@kspearrin While working on adding the Mac portion of the build to the Directory-Connector build/release pipeline, I found out there is a bug in the Github API that `hub` is using to update the draft releases. I needed to tweak to flow of both to use `actions/upload-release-asset`.

**Review by kspearrin — DISMISSED** [review]

**Review by kspearrin — APPROVED** [review]

**Pull request merged**